ntbs-service-unit-tests/TestData/MigrationDatabaseMock/notifications.csv

```diff
@@ -21,3 +21,4 @@ OldNotificationId,Source,NtbsHospitalId,EtsId,LtbrId,LtbrPatientId,GroupId,Notif
 132503,ETS,B8AA918D-233F-4C41-B9AE-BE8A8DC8BE7A,130331,,,,2015-03-31 00:00:00.000,0,,,HIVStatusKnown,Dr McGown,,Patient did not begin course of treatment under DOT,"3 Winglass Place","","Wongaton","",RG145UT,Wongus,Winford,9815779000,1981-03-24 00:00:00.000,0,38,2010,10,1,,0,,networks architect / some other occupation,"",,,,,,0,,,,Yes,1,9,,2,1,3,,2014-09-15 00:00:00.000,1,2014-10-23 00:00:00.000,,,2015-03-25 00:00:00.000,Yes,,2015-03-25 00:00:00.000,2015-03-30 00:00:00.000,0,0,StandardTherapy,,,,,Yes,1984,,,,,,Yes,0,0,0,,,,,,,No,0,0,0,,0,0,0,Yes,0,1,0,,,,,,,,,,,,,No,,,,,,,,,,
 132504,ETS,B8AA918D-233F-4C41-B9AE-BE8A8DC8BE7A,130331,,,,2015-03-31 00:00:00.000,0,,,HIVStatusKnown,Dr McGown,,Patient did not begin course of treatment under DOT,"3 Winglass Place","","Wongaton","",RG145UT,Wongus,Winford,9815779000,1981-03-24 00:00:00.000,0,38,2010,10,1,,0,,networks architect / some other occupation,"",,,,,,0,,,,Yes,1,9,,2,1,3,,2014-09-15 00:00:00.000,1,2014-10-23 00:00:00.000,,,2015-03-25 00:00:00.000,Yes,,2015-03-25 00:00:00.000,2015-03-30 00:00:00.000,0,0,StandardTherapy,,,,,Yes,1984,,,,,,Yes,1,1,1,Some€£ invalid ~characters{},,,,,,No,0,0,0,,0,0,0,Yes,0,1,0,,,,,,,,,,,,,No,,,,,,,,,,
 300999,ETS,0EEE2EC2-1F3E-4175-BE90-85AA33F0686C,132465,,,,2019-10-01 00:00:00.000,0,,,,,Robert.Greene@ntbs.phe.com,"",,,,,,Morton,Dudly,9456783125,,0,38,,10,1,,1,,,,2021-01-01 00:00:00.000,,,,,,,,,,,,,,,,,,,,,,,,,2019-10-01 00:00:00.000,,,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+301337,ETS,0EEE2EC2-1F3E-4175-BE90-85AA33F0686C,132465,,,,2019-10-01 00:00:00.000,0,,,,,Robert.Greene@ntbs.phe.com,"",,,,,,Morton,Dudly,9456783125,,0,38,,10,1,,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2020-05-06 00:00:00.000,,,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Depending on when you branched off master, you may need to add two extra fields here after you've merged, for the notifying hospital and case manager. They can both be empty, but figuring out where to put them is annoying.

Reply:
About to rebase now, not excited!